documentation/proc-pages/physics-models/fusion_reactions/beam_reactions.md

```diff
@@ -0,0 +1,392 @@
+# Neutral Beam Injection Fusion
+
+The main function called for calculating the fusion reactions produced by neutral beam injection is `beam_fusion()`
+Due to the small contribution of fusion power from the neutral beams only D-T reactions are taken into account, as D-D additions to fusion power are deemed to be negligible.
```

Comment:
Need source for DD contributions negligible or stating how small value is so others can judge and agree that it is negligible, ie 1/1000th of DT contributions

Reply:
In the main plasma the DD contribution is roughly 1000 lower that the DT one so you only get a couple of MW out from it. If you apply that to the plasma beam interaction it will be in the kW range